projects/dynamicforms/src/core/form-containers/dynamic-form/dynamic-form.component.ts

```diff
@@ -106,7 +106,7 @@ export class DynamicFormComponent implements OnInit, OnDestroy {
         .getSubmittedForm()
         .pipe(
           map(form => {
-            if (form && this.checkInvalidControls(form)) {
+            if (this.checkInvalidControls(form)) {
```

Comment:
this basically means that getSubmittedForm in case of FNOL doesn't return submitted form :/ 
Can you please check if its empty object or null?

Reply:
I think it was null, because checkInvalidControls method was skipped, but I'll check it again.

Reply:
If its null, then removing 'form' from conditions doesn't improve things because the second condition won't be fulfilled also. We can check invalid controls only if the form exists so i think that we need it.
Can you try with Object.keys(form).length !==0